components/salsa-macro-rules/src/setup_tracked_fn.rs

```diff
@@ -158,6 +158,15 @@ macro_rules! setup_tracked_fn {
 
                 const CYCLE_STRATEGY: $zalsa::CycleRecoveryStrategy = $zalsa::CycleRecoveryStrategy::$cycle_recovery_strategy;
 
+                unsafe fn maybe_update<'db>(old_pointer: *mut Self::Output<'db>, new_value: Self::Output<'db>) -> bool {
```

Comment:
```suggestion
                unsafe fn maybe_update<$db_lt>(old_pointer: *mut Self::Output<$db_lt>, new_value: Self::Output<$db_lt>) -> bool {
```

This slightly improves things, but not much